test/tools/OpenCover/OpenCover.psm1

```diff
@@ -1,5 +1,7 @@
 ﻿#region privateFunctions
 
+$script:psRepoPath = git rev-parse --show-toplevel
```

Comment:
>git rev-parse --show-toplevel [](start = 21, length = 29)

git tools are not available in Azure Functions or other automation pipelines. Please check if git tools exist before using them.